packages/hadron-build/commands/download.js

```diff
@@ -43,12 +42,18 @@ const handler = function handler(argv) {
     cli.info(`${asset.name}: download from evg bucket complete`);
   });
 
-  Promise.all(downloads).catch(abortIfError);
-};
+  return Promise.all(downloads)
+}
+
+const handler = function handler(argv) {
+  cli.argv = argv;
+  run(argv).catch(abortIfError);
```

Comment:
In the end I'm not using download because as is it would download all the packages and therefore mean we have to depend on every package task rather than just the relevant one for the platform.

I could modify download more to filter but we already have the info we need from hadron-build info anyway so I just download straight in the script.

But I think the change here is harmless and a very minor improvement anyway.